bff/src/Bff/Configuration/BffOptions.cs

```diff
@@ -139,19 +140,20 @@ public class BffOptions
     /// APIs with TokenType.User or TokenType.UserOrClient. Defaults to True. 
     /// </summary>
     public bool RemoveSessionAfterRefreshTokenExpiration { get; set; } = true;
-}
 
-/// <summary>
-/// Enum representing the style of response from the ~/bff/user endpoint when the user is anonymous.
-/// </summary>
-public enum AnonymousSessionResponse
-{
-    /// <summary>
-    /// 401 response with empty body
-    /// </summary>
-    Response401,
     /// <summary>
-    /// 200 response with "null" as the body
+    /// Implementation that allows you to check if you want to disable the anti-forgery check for a given request.
+    /// Default implementation will disable anti forgery requests for websockets, because websocket requests can't
+    /// include http headers. 
```

Comment:
```suggestion
    /// A delegate that determines if the anti-forgery check should be disabled for a given request.
    /// The default is to disable anti forgery checks for websockets, because websocket requests can't
    /// include http headers. 
```